conf/emonhub.conf

```diff
@@ -72,18 +73,48 @@ loglevel = DEBUG #(default:WARNING)
         basetopic = emonhub/
         
 [[emoncmsorg]]
+###[[Local_CMS]]
     Type = EmonHubEmoncmsHTTPInterfacer
     [[[init_settings]]]
     [[[runtimesettings]]]
         pubchannels = ToRFM12,
         subchannels = ToEmonCMS,
-        url = http://emoncms.org
-        apikey = xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
-        # data_send_interval = 20 #seconds
-        # status_send_interval = 60 
+	#url = http://api.gle.solar
+	#url =http://sesh-dev.westeurope.cloudapp.azure.com:3030/
+	url = http://api.gle.solar
+        #apikey = nyange2016
         senddata = 1
-        sendstatus = 0
-        
+        sendstatus = 1
+    	site_id = 5
+
+
+#[[BMV]]
```

Comment:
these comments look odd? We shuold leave the interfacer as a tempalte commented out.

Reply:
I don` understand what you mean, BMV and this data logger use the same serial port, one of them need to be commented out and here BMV is commented